### DISCUSSION TITLE
Screen URLs

### DISCUSSION THREAD

**Discussion by philwakefield** [Budibase/budibase#1464] [Help] [answered]
I am not sure if I am misunderstanding something but the only way I can get a card to link to another screen is using the following structure:

app_c3f79118b36541d2976fc8ad6d32fde3#/names/{{ New Repeater.Names._id }}

I would have thought the 'app_c3f79118b36541d2976fc8ad6d32fde3#' part would not be required but if I exclude it I get:
{"message":"missing","status":404}

Thanks for the help.

Phil

**Comment by mjashanks** [comment, marked as answer]
Hey Phil. Try `#/names/{{ New Repeater.Names._id }}` - relative path, with leading `#`.

**Reply by philwakefield** [reply]
Thanks that works perfectly!

**Comment by aptkingston** [comment]
Hi @philwakefield . Unfortunately there are currently 2 components which have a bug with links - `Stacked List` and `Horizontal Card`. You're right that you should not need to enter the app ID into the URL. Normally all you need is `/names/something` for example. This has been fixed but it hasn't been released yet. Try what @mjashanks suggested, hopefully that works for now until the fix is released. Like I said this only affects these 2 components. You can always use other components or build a card yourself and use the `Link` component if you'd like.

**Reply by philwakefield** [reply]
Ah I did wonder - @mjashanks solution worked nicely.
